MSAL/src/public/MSALResult.h

```diff
@@ -40,13 +40,13 @@
 #pragma mark - Token response
 
 /** The Access Token requested. */
-@property (readonly, nonnull) NSString *accessToken;
+@property (readonly, nullable) NSString *accessToken;
```

Comment:
Bit concerned here as they were nonnulll earlier and could cause crashes without null check

Reply:
Do you have any suggestions? We can return an empty string, but that feels unnatural.

Reply:
As discussed offline, we can resolve this